api/lib/domain/events/handle-pix-plus-certification.js

```diff
@@ -0,0 +1,42 @@
+const bluebird = require('bluebird');
+const { checkEventType } = require('./check-event-type');
+const CertificationScoringCompleted = require('./CertificationScoringCompleted');
+const PixPlusCertification = require('../models/PixPlusCertification');
+const { ReproducibilityRate } = require('../models/ReproducibilityRate');
+
+const eventType = CertificationScoringCompleted;
+
+async function handlePixPlusCertifications({
+  event,
+  domainTransaction,
+  badgeAcquisitionRepository,
+  certificationAssessmentRepository,
+  partnerCertificationRepository,
+  targetProfileWithLearningContentRepository,
+}) {
+  checkEventType(event, eventType);
+  const userId = event.userId;
+  const certificationCourseId = event.certificationCourseId;
+  const certificationAssessment = await certificationAssessmentRepository.getByCertificationCourseId({ certificationCourseId, domainTransaction });
+  const date = certificationAssessment.createdAt;
+
+  const certifiableBadges = await badgeAcquisitionRepository.findCertifiable({ userId, date, domainTransaction });
+  const pixPlusCertifications = await bluebird.mapSeries(certifiableBadges, async (certifiableBadge) => _computePixPlusCertification(certifiableBadge, event, certificationAssessment, targetProfileWithLearningContentRepository));
+  await partnerCertificationRepository.saveBatch({ partnerCertifications: pixPlusCertifications, domainTransaction });
+}
+
+async function _computePixPlusCertification(certifiableBadge, event, certificationAssessment, targetProfileWithLearningContentRepository) {
+  const certificationCourseId = event.certificationCourseId;
+  const { badgeKey: certifiableBadgeKey, targetProfileId } = certifiableBadge;
```

Comment:
Proposition, pour documenter le type de l'objet attendu :

```suggestion
async function _computePixPlusCertification(badgeAcquisition, event, certificationAssessment, targetProfileWithLearningContentRepository) {
  const certificationCourseId = event.certificationCourseId;
  const { badgeKey: certifiableBadgeKey, targetProfileId } = badgeAcquisition;
```

Reply:
(et idem au dessus ?)